components/form-control/demo/demo.html

```diff
@@ -72,14 +73,14 @@
         <h2>GET</h2>
         <gameface-form-control onload="onRequestFinished(event);" action="http://localhost:3000/user" method="get">
             <div class="form-element">
-                <span>User name:</span><input name="username" value="name" type="text" />
+                <span>User name:</span><input name="username" value="name" type="text" required />
             </div>
             <div class="form-element">
-                <span>Password:</span><input name="password" value="pass" type="password" />
+                <span>Password:</span><input minimum="5" name="password" value="pass" type="password" required/>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password#minlength

```suggestion
                <span>Password:</span><input minlength="5" name="password" value="pass" type="password" required/>
```

Reply:
Changed everywhere.